docs/static/settings/monitoring-settings-legacy.asciidoc

```diff
@@ -28,12 +28,18 @@ Logstash metrics must be routed through your production cluster. You can specify
 a single host as a string, or specify multiple hosts as an array. Defaults to
 `http://localhost:9200`.
 
-NOTE: If your Elasticsearch cluster is configured with dedicated master-eliglble
+NOTE: If your Elasticsearch cluster is configured with dedicated master-eligible
 nodes, Logstash metrics should _not_ be routed to these nodes, as doing so can
 create resource contention and impact the stability of the Elasticsearch
 cluster. Therefore, do not include such nodes in
 `xpack.monitoring.elasticsearch.hosts`.
 
+`xpack.monitoring.elasticsearch.proxy`::
+
+The monitoring {es} instance and monitored Logstash can be separated by a proxy.
+To enable Logstash to connect a proxed {es}, set this value to the URI of the intermediate proxy.
```

Comment:
```suggestion
To enable Logstash to connect to a proxied {es}, set this value to the URI of the intermediate proxy.
```

Reply:
@andsel  I didn't get my suggestion quite right before. Please consider using the suggested tweak. 

Do you think it would be helpful to say, "... set this value to the URI of the intermediate proxy using the standard URI format, <proto>://<path>"?   I trust you judgement on whether or not that should be added.